csrc/scheduler/matmul.cpp

```diff
@@ -437,6 +436,186 @@ void scheduleProlog(TensorView* shared_mem_tv, const MatmulParams& params) {
           .propagateParallelType());
 }
 
+//! swizzle the shared mem data layout using a same method in prologSwizzle.
+//! The shift parameter is added for the transform of MMA results to skip the
+//! K axis and will skip the actual swizzle. This is to ensure a same transform
+//! history between the MMA result tensor and the epilogue shared memory tensor
+//! so the corresponding domains of these two tensors can be mapped. This
+//! function may be merged with prologSwizzle as they are using a same method.
+int epilogSwizzle(
+    TensorView* shared_mem_tv,
+    const MatmulParams& params,
+    const int shift = 0) {
+  check_concrete_static_dim(shared_mem_tv->axis(-2 - shift));
+  check_concrete_static_dim(shared_mem_tv->axis(-1 - shift));
+  // Extract the constant sizes of the swizzled tile, e.g. 128 x 128
+  const int tile_size_x =
```

Comment:
How about switching to `int64_t` from `int`? Similarly how it has been done in the `prologSwizzling`.

Reply:
changed.